migrations/20200707221810-mandatory-patient-fields.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.changeColumn("patients", "barcodeValue", {
+          type: Sequelize.BIGINT,
+          allowNull: false,
+        }),
```

Comment:
turns out the correct way to use enums (probably) is appending `{ transaction: t }` as an argument to the `queryInterface` call, so make sure to do that for all 4 calls you have
```suggestion
        queryInterface.changeColumn("patients", "barcodeValue", {
          type: Sequelize.BIGINT,
          allowNull: false,
        }, { transaction: t }),
```